.github/scripts/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Can we rename this filename to something clearer, i.e. build-aws.sh

Reply:
Done! https://github.com/ntampakas/tlsn/commit/6290dc85af85eb4857edd9ff457271a4a6d63e12